tool_collections/fastx_toolkit/fasta_clipping_histogram/fasta_clipping_histogram.xml

```diff
@@ -1,18 +1,25 @@
-<tool id="cshl_fasta_clipping_histogram" name="Length Distribution" version="1.0.0">
+<tool id="cshl_fasta_clipping_histogram" name="Length Distribution" version="@VERSION@">
```

Comment:
I wonder if this would cause problems, unless the version lineage code recognizes that 0.0.14 > 1.0.0